examples/azure-service-mesh/security/cert-manager-letsencrypt/readme.md

```diff
@@ -0,0 +1,269 @@
+# Cert-Manager Let's Encrypt Integration with Istio-based service mesh add-on
+
+This document contains instructions on how to integrate Istio-based service mesh add-on for AKS with cert-manager and obtain letsencrypt certificates for setting up secure ingress gateways. 
+
+## Objectives
+* Deploy bookinfo demo app, expose a secure HTTPS service using simple TLS.
+* Demonstrate HTTPS connections for Azure Service Mesh workloads using Cert-Manager and letsencrypt as the certificate authority.
+
+## Before you begin
+* Install Install Azure Servicemesh.
+* Enable external ingressgateway.
+* Enable sidecar injection on the default namespace. 
+
+```shell
+az aks mesh enable -g <rg-name> -n <cluster-name>
+az aks mesh enable-ingress-gateway -g <rg-name> -n <cluster-name> --ingress-gateway-type external
+revision=$(az aks show --resource-group <rg-name> --name <cluster-name> --query 'serviceMeshProfile.istio.revisions[0]' -o tsv)
+kubectl label namespace default istio.io/rev=$revision
+```
```

Comment:
Ah I see you have commands listed below the bullet points. the section above said before you begin. would recommend interleaving these commands in between those points to avoid confusion.